### DISCUSSION TITLE
Add a new project in the Related projects

### DISCUSSION THREAD

**Discussion by cosminadrianpopescu** [neovim/neovim#30043] [General]
Hello all,

I created [azul](https://github.com/cosminadrianpopescu/azul), a full terminal multiplexer (not a neovim plugin) based on neovim terminal capabilities. It has floating panels, splits, themes, sessions and everything you could expect from a terminal multiplexer. In my opinion it can rival with [tmux](https://github.com/tmux/tmux) or [zellij](https://zellij.dev). 

I use it as a daily driver for all my development activities in all my environments (work and personal) since several months. 

Can it be included in the related projects page of `neovim` in the `non-plugin` category?

**Comment by zeertzjq** [comment]
It's included now.
